docs/src/pages/components/authenticator/formfields/form-fields.web.mdx

```diff
@@ -0,0 +1,184 @@
+import {
+  Authenticator,
+  CheckboxField,
+  useAuthenticator,
+} from '@aws-amplify/ui-react';
+
+import { Example } from '@/components/Example';
+import { Fragment } from '@/components/Fragment';
+
+### Sign Up Field Order
+
+The authenticator allows a custom order of sign up fields on the Sign Up page.
+
+Order is determined by a `formFields` prop that is passed into the `authenticator` component.
+This prop will have the `signUp` key with a list of all the input fields. Each input field will
+have an `order` key that specifies which order it will be displayed on the page.
+
+In the example below the order will be set as `email`, `family_name`, `birthdate`, `preferred_username`, `password` and finally `confirm_password`.
+
+**Note:** Fields that do not have a `order` key will be displayed at the bottom of the Sign Up page. The `order` key can also be combined with other form field updates.
+
+<Fragment>
+  {({ platform }) => import(`./orderformfields/form-fields.${platform}.mdx`)}
+</Fragment>
+
+<Example>
+  <Authenticator
+    initialState={'signUp'}
+    loginMechanisms={['email']}
+    signUpAttributes={['birthdate', 'family_name', 'preferred_username']}
+    formFields={{
+      signUp: {
+        email: {
+          order:1
+        },
+        family_name: {
+          order: 2
+        },
+        preferred_username: {
+          order: 4
+        },
+        birthdate: {
+          order: 3
+        },
+        password: {
+          order: 5
+        },
+        confirm_password: {
+          order: 6
+        }
+      },
+    }}
+
+/>
+
+</Example>
+
+### Form Field Customization
+
+The Authenticator allows for customization of the form input fields throughout the Authenticator.
+
+#### Updating labels, placeholders, required fields and showing labels.
+
+You can customize any label, placeholder, set a field as required or not required, and hide or show labels by configuring the `formFields` props, and passing it into the `authenticator` component.
+To use this feature create a `formFields` prop and include the component name as a key.
+Inside that object you can list all the inputs you'd like to change by their name.
+
+The following example customizes the Sign In page by:
+
+- Updating the placeholder with `placeholder`.
+- Setting required to true with `required`, which is also the default for this field.
+- Updating the label text with `label`.
+- Show the label using `labelHidden` set to `false`.
+
+**Note:** On the sign in page the input name is always `username`, regardless of the login mechanism type.
+
+<Fragment>{({ platform }) => import(`./form-fields.${platform}.mdx`)}</Fragment>
+
+<Example>
+  <Authenticator
+    loginMechanisms={['email']}
+    formFields={{
+      signIn: {
+        username: {
+          labelHidden: false,
+          placeholder: 'Enter Your Email Here',
+          required: true,
```

Comment:
This should be `isRequired`. The reason `required` may also work is because it is passed on to the DOM element. But I think we should use `isRequired` for consistency within the docs: 
https://ui.docs.amplify.aws/components/textfield#required-fields

Reply:
Vue and Angular don't use the `textField` component so the internal components all just take in a prop named `required`. So I think we should just keep this as `required`.

Reply:
Can we just change it for React docs then? I don't want our docs to be showing use of both `isRequired` and `required`.

Reply:
ok updated to `isRequired` in d1f48302035dec842dc4d23753a3ea379de129fc

Reply:
And 3c90655764811d36b88cdcb2b7cd8f3e40ac64e